### PR TITLE
MTTL-665 Added SNS topic to terraform files

### PR DESCRIPTION
**Pull request by evak6979**
## Link to JIRA ticket

https://hackney.atlassian.net/browse/MTTL-665

### *What is the problem we're trying to solve*

Create an SNS topic

### CODE REVIEW

File: terraform/development/main.tf [hunk 1]
```diff
@@ -36,3 +36,9 @@ terraform {
     key     = "services/person-api/state"
   }
 }
+
+resource "aws_sns_topic" "user_updates" {
+  name                        = "Person"
```
Comment (mehdi-kimakhe-amido) [inline terraform/development/main.tf:41]: @evak6979 For FIFO topics, the name should end with ".fifo" I believe

File: terraform/production/main.tf [hunk 2]
```diff
@@ -36,3 +36,9 @@ terraform {
     key     = "services/person-api/state"
   }
 }
+
+resource "aws_sns_topic" "user_updates" {
+  name                        = "Person"
```
Comment (mehdi-kimakhe-amido) [inline terraform/production/main.tf:41]: @evak6979 For FIFO topics, the name should end with ".fifo" I believe

File: terraform/staging/main.tf [hunk 3]
```diff
@@ -36,3 +36,9 @@ terraform {
     key     = "services/person-api/state"
   }
 }
+
+resource "aws_sns_topic" "user_updates" {
+  name                        = "Person"
```
Comment (mehdi-kimakhe-amido) [inline terraform/staging/main.tf:41]: @evak6979 For FIFO topics, the name should end with ".fifo" I believe

File: terraform/development/main.tf (hunk 1 above)
Comment (evak6979) [reply terraform/development/main.tf:41]: I thought so too but didn't see it as part of the ticket.

**Review by mehdi-kimakhe-amido — APPROVED** [review]

**Review by amidomichael — APPROVED** [review]

**Pull request merged**